### PR TITLE
Koding Issue #9329 - ReadMe Preview and Credential Show Preview Modal…

### LINKED ISSUE #9329: ReadMe Preview and Credential Show Preview Modal's styling needs to be fixed
As shown in screenshots, these modal's styling is needs to fixed.
/cc: @ftasdelen, @diclee 
## Design changes

[New Modal Styling 04.pdf](https://github.com/koding/koding/files/545249/New.Modal.Styling.04.pdf)
## Acceptance Criteria

The pull request must
- [ ] satisfy all requirements
- [ ] provide screenshot for each modal
- [ ] satisfy [coffeescript style guideline](https://github.com/koding/styleguide-coffeescript)
- [ ] make sure all other windows are not affected with this changes.

If something is not clear, discuss it below.

### PR DESCRIPTION
**Pull request by royalfl**
ReadMe Preview and Credential Show Preview Modal's styling needs to be fixed #9329
## Description

I have fixed ReadMe Preview and Credential Show Preview Modal's styling
## Motivation and Context

https://github.com/koding/koding/issues/9329
## How Has This Been Tested?

Please checkout with following url
http://developer.jacksmith89129.koding.team:8090/
## Screenshots (if appropriate):

screenshot from 2016-10-24 18 06 17
screenshot from 2016-10-24 18 06 37
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by hakankaradis** [comment]
@jack89129 LGTM but we need to make sure this changes will not affect to other modals that are using same styl file.

File: client/app/lib/views/credentiallist/accountcredentiallist.coffee [hunk 1]
```diff
@@ -24,14 +24,22 @@ module.exports = class AccountCredentialList extends KodingListView
   showCredential: (options = {}) ->
 
     { credential, cred } = options
+    
+    content = "<p><pre><code>" + cred + "</code></pre></p>"
```
Comment (hakankaradis) [inline client/app/lib/views/credentiallist/accountcredentiallist.coffee]: https://github.com/koding/styleguide-coffeescript#use-string-interpolations-instead-of-string-concatenation

`content = "<p><pre><code>#{cred}</code></pre></p>"`

File: client/app/lib/components/contentModal/styl/contentModal.styl [hunk 2]
```diff
@@ -275,11 +275,18 @@ main.main-container
...
-        overflow            auto
+      /* margin-top            10px */
+      /* max-width             450px */
+      padding                   10px 30px
+      
+      pre
+        margin                  0px
+        
+        code
+            overflow            hidden
```
Comment (hakankaradis) [inline client/app/lib/components/contentModal/styl/contentModal.styl]: indentation, must be 2 spaces.

**Comment by jack89129** [comment]
This changes will not affect to other modals. The ".kdscrollview .markdown-content" style are using for only ReadMe Preview and Credential Show Preview Modal.

**Comment by jack89129** [comment]
I have fixed the page's horizontal scroll bar doesn't work.

screenshot from 2016-10-24 19 45 14

**Pull request closed without merging**